src/gait/linear_client.py

```diff
@@ -0,0 +1,124 @@
+from gql import gql, Client
+from gql.transport.requests import RequestsHTTPTransport
+from gql.transport.exceptions import TransportQueryError
+from dotenv import load_dotenv
+import os
+
+class LinearClient:
+    def __init__(self, test_mode: bool = False):
+        self.test_mode = test_mode
+        if not test_mode:
+            load_dotenv(override=True)
+            self._init_client()
+            
+    def _init_client(self):
+        """Initialize Linear GraphQL client"""
```

Comment:
Let's remove this comment

Reply:
For more context: https://alexkondov.com/comments-the-good-the-bad-the-ugly/

Basically in general we want to avoid "what" comments and utilize comments to document "why" (when necessary)

The reason for this is that the code itself already documents what is going on. Then when you update the code at some point the comment will stay the same and then stop representing what it should be doing.